internal/rbd/rbd_util.go

```diff
@@ -175,11 +175,6 @@ func (rs *rbdSnapshot) String() string {
 	return fmt.Sprintf("%s/%s@%s", rs.Pool, rs.RbdImageName, rs.RbdSnapName)
 }
 
-// imageSpec returns the image-spec (pool/image) format of the snapshot
```

Comment:
Why does commit 62aadd543269f07ab8d934274fc2d62b64259eb3 remove `imageSpec()`? It was not used in the rest of the change of the commit.

Reply:
Imagespec is not used anymore, The relevant changes are removed in final commit you want this change to be part of same commit?

Reply:
corrected